diff -urN mate-panel-1.26.3.orig/libmate-panel-applet/Makefile.in mate-panel-1.26.3/libmate-panel-applet/Makefile.in
--- mate-panel-1.26.3.orig/libmate-panel-applet/Makefile.in	2023-05-02 14:46:00.000000000 +0000
+++ mate-panel-1.26.3/libmate-panel-applet/Makefile.in	2023-08-28 20:59:14.718664766 +0000
@@ -1032,7 +1032,7 @@
 			--eprod "GType @enum_name@_get_type (void);\n" \
 		$(mate_panel_applet_enum_headers) > $@
 $(noinst_DATA): $(applet_in_files)
-	$(AM_V_GEN) $(MSGFMT) --desktop --keyword Name --keyword Description --template $< -d $(top_srcdir)/po -o $@
+	$(AM_V_GEN) $(MSGFMT) --desktop --keyword=Name --keyword=Description --template $< -d $(top_srcdir)/po -o $@
 
 ###############################
 # Introspection generation
